src/utils.js

```diff
@@ -20,6 +20,17 @@ export function appIconUrl(app) {
     : null
 }
 
+export function isElectron() {
+  // See https://github.com/electron/electron/issues/2288
+  if (
+    typeof navigator === 'object' &&
+    typeof navigator.userAgent === 'string' &&
+    navigator.userAgent.indexOf('Electron') >= 0
+  ) {
+    return true
+  }
```

Comment:
We should return `false` instead of `undefined`, what about returning the expression directly?